### PR TITLE
deployment of SDK docs

### PR DESCRIPTION
**Pull request by erohmensing**
**Proposed changes**:
- ...

**Status (please check what you already did)**:
- [ ] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa-sdk/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa-sdk#code-style) for instructions)

### CODE REVIEW

**Comment by m-vdb** [comment]
@erohmensing you need a file named `docs/static/_redirects`, like we do here: https://github.com/RasaHQ/rasa/blob/master/docs/static/_redirects

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -14,10 +13,9 @@ let legacyVersions = [];
 module.exports = {
   title: 'Rasa SDK Documentation',
   tagline: 'Rasa SDK Documentation',
-  // TODO: is it needed?
-  url: 'http://something.com',
-  baseUrl: '/docs/rasa/rasa-sdk/',  //TODO: seems fake
-  onBrokenLinks: 'throw',
+  url: 'https://rasa.com',
+  // FIXME: remove "next/" when releasing
+  baseUrl: '/docs/rasa-sdk/next/',
```
Comment (m-vdb) [inline docs/docusaurus.config.js:18]: you can remove the next right away, since this URL doesn't exist at all today

File: docs/package.json [hunk 2]
```diff
@@ -18,13 +18,13 @@
...
+    "@docusaurus/plugin-sitemap": "^2.0.0-alpha.61",
+    "@docusaurus/theme-classic": "^2.0.0-alpha.61",
     "@mdx-js/react": "^1.5.8",
-    "clsx": "^1.1.1",
     "react": "^16.8.4",
-    "react-dom": "^16.8.4",
-    "remark-collapse": "^0.1.2",
-    "remark-sources": "^1.0.3"
+    "react-dom": "^16.8.4"
   },
```
Comment (erohmensing) [inline docs/package.json:28]: I see what you mean now about the dependencies! I shouldnt have added them before, it just makes it more work to try to get the deployment working. We will almost certainly use them, which is why i've left the comments in the docusaurus config to remind me where to put them back in, but i'll update those on the content branch: https://github.com/RasaHQ/rasa-sdk/pull/254/files

Sorry for the confusion on the last one 😄

**Review by m-vdb — CHANGES_REQUESTED** [review]
Nice work @erohmensing ! a couple of comments to address before it's mergeable :)

**Comment by erohmensing** [comment]
erp, forgot to re-request a review!

**Review by m-vdb — APPROVED** [review]
🥇

**Pull request merged**